tmexp/__main__.py

```diff
@@ -163,14 +182,14 @@ def get_parser() -> argparse.ArgumentParser:
     create_bow_parser.add_argument(
         "--min-word-frac",
         help="Words occuring in less then this percentage of all documents are removed,"
-        " default to 2 %.",
+        " default to 2 %%.",
         type=float,
         default=0.02,
     )
     create_bow_parser.add_argument(
         "--max-word-frac",
         help="Words occuring in more then this percentage of all documents are removed,"
-        " default to 80 %.",
+        " default to 80 %%.",
```

Comment:
sorry I missed it earlier but we should use `%(default)s` to avoid out-of-sync help texts

Reply:
done